neqo-transport/src/qlog.rs

```diff
@@ -543,7 +543,7 @@ impl From<Frame<'_>> for QuicFrame {
             Frame::HandshakeDone => Self::HandshakeDone,
             Frame::AckFrequency { .. } => Self::Unknown {
                 frame_type_value: None,
-                raw_frame_type: frame.get_type(),
+                raw_frame_type: frame.get_type().into(),
                 raw: None,
             },
```

Comment:
Interesting.  I guess we can look at whether qlog supports the frame when we get around to updating that spec.